eng/Signing.props

```diff
@@ -43,6 +43,10 @@
     <FileExtensionSignInfo Include=".otf" CertificateName="None" />
     <FileExtensionSignInfo Remove=".js" />
     <FileExtensionSignInfo Include=".js" CertificateName="None" />
+    
+    <!-- Ensure that azl files get the mariner cert -->
+    <AzureLinuxRPM Include="$(ArtifactsPackagesDir)**/*-azl-*.rpm" />
+    <FileSignInfo Include="@(AzureLinuxRPM->'%(Filename)%(Extension)')" CertificateName="LinuxSignMariner" />
```

Comment:
I see that you are repeating this pattern here and in aspnetcore. Should this be handled centrally in arcade's signing infra?

Reply:
Possibly? Let me give it a try. It is a little awkward.